src/zc/buildout/buildout.txt

```diff
@@ -1088,10 +1088,10 @@ Extending sections (macros)
 ---------------------------
 
 A section (other than the buildout section) can extend one or more
-other sections using the ``<`` option.  Options from the referenced
```

Comment:
Describing this as the ``<`` option is correct; the ``=`` comes from the ``option = value`` structure.